### ISSUE TITLE
"Maximum call stack size exceeded" on npm start

### ISSUE THREAD

**Issue by jacklenehan** [appnexus/lucid#884]
```
> lucid-ui@2.29.1 start /Users/jlenehan/Dropbox/code/lucid
> npm run dev


> lucid-ui@2.29.1 dev /Users/jlenehan/Dropbox/code/lucid
> gulp docs-generate && webpack-dev-server --inline --hot --content-base src/docs/

[12:30:56] Using gulpfile ~/Dropbox/code/lucid/gulpfile.js
[12:30:56] Starting 'docs-generate'...
[12:30:57] 'docs-generate' errored after 1.18 s
[12:30:57] RangeError: Maximum call stack size exceeded
    at isObjectLike (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:11786:26)
    at isSymbol (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:12173:10)
    at isKey (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:6345:28)
    at castPath (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:4470:14)
    at baseGet (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:3040:14)
    at get (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:13115:49)
    at /Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:3589:24
    at baseFindIndex (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:823:11)
    at findIndex (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:7275:14)
    at Function.find (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:5098:21)
    at interceptor (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:16970:35)
    at Function.thru (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:8812:14)
    at /Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:4379:28
    at arrayReduce (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:704:21)
    at baseWrapperValue (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:4378:14)
    at LodashWrapper.wrapperValue (/Users/jlenehan/Dropbox/code/lucid/node_modules/lodash/lodash.js:9067:14)
```

**Comment by jacklenehan** [comment]
Seems to be an error starting in `anx-docgen/index.js:109`

**Comment by jacklenehan** [comment]
Addressed in #895
